src/app/view/perspectives/post-it/document/post-it-document-cell/post-it-document-cell.component.html

```diff
@@ -0,0 +1,24 @@
+<span id="{{perspectiveId}}#{{key}}#{{column}}#{{row}}"
+      [attr.tabindex]="index * 1000 + row * 2 + column"
+      [title]="model" class="d-flex"
+      (keydown.ArrowLeft)="selectionHelper.moveLeft()"
+      (keydown.ArrowRight)="selectionHelper.moveRight()"
+      (keydown.ArrowUp)="selectionHelper.moveUp()"
+      (keydown.ArrowDown)="selectionHelper.moveDown()"
+      (keydown.F2)="selectionHelper.focusToggle(true)"
+      (keydown.enter)="selectionHelper.focusToggle(true)"
+      (keydown.backspace)="onRemove()"
+      (keydown.delete)="onRemove()"
+      (focus)="onFocus()">
```

Comment:
If you want to reduce the number of DOM elements, you can set all these attributes and event listeners on the component element itself by using `@HostBinding` and `@HostListener` decorators.